sync-core/src/main/java/com/cloudant/sync/query/OperatorExpressionNode.java

```diff
@@ -0,0 +1,32 @@
+//  Copyright (c) 2014 Cloudant. All rights reserved.
```

Comment:
Updated all copyright notices on classes written in 2015.  Done in https://github.com/cloudant/sync-android/commit/cf527fd540554a34338857f179c44fa358444ad3